packages/prism/package.json

```diff
@@ -10,6 +10,7 @@
     "watch": "microbundle watch --no-compress -o dist"
   },
   "dependencies": {
+    "gatsby-design-tokens": "^1.0.0",
```

Comment:
Thanks for working on this! I don't think we want to use Gatsby brand colors for something like this. Instead, the Prism component should be themeable like the rest of the library. The `title` prop could be wrapped in a single `<div>` with a [variant](https://theme-ui.com/guides/variants/) to allow styles to be applied in the `theme` object – sorry that we don't have a lot of examples/docs around this stuff yet

Reply:
@jxnblk So are you saying do something like this?

```js
{title && (
  <div sx={{ variant: 'prism.code-title' }}>
      {title}
  </div>
 )}
```

And then have the user set the `prism.code-title` variant to whatever they want?

Reply:
Yes, something like that, though I'd avoid using `-` in a JS key name based on common conventions

Reply:
okay @jxnblk ... I'll call it `prism.codeTitle`